### DISCUSSION TITLE
Filament for Laravel REST API

### DISCUSSION THREAD

**Discussion by DevMushref** [filamentphp/filament#5873] [Help]
I started to build a REST API in Laravel to be used in a mobile application. I followed this [tutorial](https://www.youtube.com/watch?v=xvqPEEpRBJ4)

I understood how API works and ready to go, but how to implement this in filament rules? It looks like I obviously shouldn't make controllers for every filament resource and create `store()`, `delete()` methods just for the sake of API requests.

Does Filament have anything that supports this need? POSTing and GETting data though APIs?

**Comment by danharrin** [comment]
Filament doesn't build an API, you built that yourself. Filament is separate
